src/Modules/Fields/EditMeshBoundingBox.h

```diff
@@ -29,6 +29,7 @@
 #ifndef MODULES_FIELDS_EDITMESHBOUNDINGBOX_H
 #define MODULES_FIELDS_EDITMESHBOUNDINGBOX_H
 
+#include "Graphics/Widgets/BoundingBoxWidget.h"
```

Comment:
```suggestion
#include <Graphics/Widgets/BoundingBoxWidget.h>
```
Please change all #includes to use <> instead of ""